### ISSUE TITLE
Deleting and ignoring search terms

### ISSUE THREAD

**Issue by ajtruckle** [Mark-met-een-K/wp-search-insights#187]
Hi

Thanks for the latest version. But I am still somewhat confused:

![Log](https://user-images.githubusercontent.com/7985331/173522060-c42fc3ae-73f4-4266-ab1b-d0c8f12aab17.jpg)

1/ Why is it that when I select one entry and say "Delete and Ignore" that it then doesn't apply this to the rest of the log? I have to do them one by one. It would be nice if it offered to purge all instances.

2./ Why is it that any entry I tell it to "delete and ignore" that it is not added to the "ignore" list in the settings? There seems to be no way to see what these terms are that I can chosen to ignore so that I could undo that for one of them if I wanted to.

Thanks.

**Comment by Mark-met-een-K** [comment]
Agreed. I've added this to the to-do list. Results should disappear from the table after refresing the page too
